examples/webapp/db/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:latest
+
+ENV RQLITE_VERSION=7.3.1
+
+RUN apk update && \
+    apk --no-cache add curl tar bash && \
+    curl -L https://github.com/rqlite/rqlite/releases/download/v${RQLITE_VERSION}/rqlite-v${RQLITE_VERSION}-linux-amd64-musl.tar.gz -o rqlite-v${RQLITE_VERSION}-linux-amd64-musl.tar.gz && \
+    tar xvfz rqlite-v${RQLITE_VERSION}-linux-amd64-musl.tar.gz && \
+    cp rqlite-v${RQLITE_VERSION}-linux-amd64-musl/rqlited /bin && \
+    cp rqlite-v${RQLITE_VERSION}-linux-amd64-musl/rqlite /bin && \
+    rm -fr rqlite-v${RQLITE_VERSION}-linux-amd64-musl rqlite-v${RQLITE_VERSION}-linux-amd64-musl.tar.gz && \
+    apk del curl tar
+
+RUN mkdir -p /rqlite/file
+
+EXPOSE 4001 4002
+COPY run_rqlite.sh /bin/run_rqlite.sh
```

Comment:
As far as I understand, it is important to put this in `/bin` and not in a VOLUME.
This deserves a comment (especially that we use VOLUMEs everywhere else).

Reply:
well, _generally_ everywhere in golem, you cannot put stuff into VOLUMES within an image since VOLUMES are mapped to empty directories and their contents inside the image are not preserved...

but here, we're not using any volume anyway

Reply:
OK, my bad.
I remember you saying that something-something-filesystem was important, and I already forgot the details, though we might want to have a comment regarding them somewhere.